tests/data/cairo1_contracts.py

```diff
@@ -0,0 +1,522 @@
+CAIRO_BINDINGS_CONTRACT_ENUM = """\
```

Comment:
I don't mind where they are put and read honestly.

Yes, they are copied and it's done on purpose because it's better (easier) to have exactly the same contracts here and in the cairo repository so if anything goes wrong we can debug it. At least for now, they're constantly changing different features and it's really important to be sure that we do as many things as possible exactly the same as on their side.

> Testing the compiler implementation is not something we'd like to do.

I think it doesn't matter that much what contracts here we're going to compile, what matters is that exactly the same contracts work with the original compiler. That's how we don't test the compiler itself but the bindings.

Reply:
As for the place for the contracts, this is done similarily to places like these:
```
tests/integration/data_transformer/contracts.py
tests/data/contracts.py

```

Reply:
[move contracts' contents to separate files](https://github.com/software-mansion/protostar/pull/1316/commits/7a524ce16343cda8c9747fc3bbe5449a31023aa0)